federation-2/router-bridge/js-src/api_schema.ts

```diff
@@ -7,12 +7,29 @@ import {
 
 import { buildSupergraphSchema } from "@apollo/federation-internals";
 
-export function apiSchema(sdl: string): ExecutionResult<String> {
+export interface ApiSchemaOptions {
+  graphqlValidation?: boolean;
```

Comment:
Should we call it `enabledGraphqlValidation` ? no strong opinion here, just a thought

Reply:
I think just using the noun is consistent with other options in the project, eg. `autoFragmentization: true` in `QueryPlanOptions`.